plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -137,6 +157,47 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
             }
     }
 
+    private fun processFunction(symbol: FirFunctionSymbol<*>, signature: FullNamedFunctionSignature): UserFunctionEmbedding =
+        UserFunctionEmbedding(processCallable(symbol, signature))
```

Comment:
Is there value in keeping this method if it's just one call to a constructor?

Reply:
Good point, removed.  I suspect we'll end up doing more with it, but we can reintroduce it when that happens.